modules/nf-core/checkm/lineagewf/main.nf

```diff
@@ -0,0 +1,47 @@
+process CHECKM_LINEAGEWF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::checkm-genome=1.2.1" : null)
```

Comment:
Seems the CheckM modules need to be updated as they still contain `params.enable_conda`